lib/utils/parse.ex

```diff
@@ -132,7 +149,7 @@ defmodule StarkInfra.Utils.Parse do
     end
   end
 
-  defp verify_signature(user, content, signature, cache_pid, _counter) do
+  def verify_signature(user, content, signature, cache_pid, _counter) do
```

Comment:
eu n sei a diferença entre def e defp, só to pontuando caso tenha mudado sem querer